docs/UserGuide.adoc

```diff
@@ -132,6 +132,22 @@ Deletes the 2nd person in the address book.
 `delete 1` +
 Deletes the 1st person in the results of the `find` command.
 
+=== Removing a tag: `remove`
+
+Removes the specific tag from the address book. +
+Format: `remove TAG`
+
+****
+* Removes the tag with the specific name `TAG`.
+* The name refers to the name of tag, or more specifically, the words in the colored rectangles under the person's name.
```

Comment:
Reference to name unclear. I suggest just saying "Removes the tag `TAG`."
The "words in coloured rectangles" part seems oddly specific; presumably the user knows what tags are after adding them and seeing them show up in the UI,